internal/policies/mount/mount.go

```diff
@@ -156,6 +176,135 @@ func (m *Manager) applyUserMountsPolicy(ctx context.Context, username string, en
 	return nil
 }
 
+func (m *Manager) applySystemMountsPolicy(ctx context.Context, machineName string, entry entry.Entry) (err error) {
+	decorate.OnError(&err, "error when applying mount policy to machine %q", machineName)
+
+	log.Debugf(ctx, "Applying mount policy to machine %q", machineName)
+
+	var failures []string
+
+	prevUnits := m.currentUnits()
+	newUnits, err := createUnits(parseEntryValues(entry))
```

Comment:
I feel like we need to talk more about this... I do remember one conversation where we agreed that, if something failed during the process, we should return an error (even though it continued until the end).

Reply:
We took that offline (this covers also all the other errors related comments). The goal is to:
- error out in the configuration generation, due to bad user input or by the system itself (creating units, daemon-reload, enable…)
- only prints warning during execution ("start/stop") as we are not responsible for it (and during next boot, it will be done automatically by systemd)